### DISCUSSION TITLE
Groops Scripts Demonstrated in The YouTube Videos

### DISCUSSION THREAD

**Discussion by Waley04** [groops-devs/groops#83] [Ideas]
Greetings GROOPS Developers,

Thanks for the new GROOPS version and the resources to support learners and users. While one can manage to watch and create the same, I was wondering if you could also add the scripts you demonstrated in the YouTube videos in your repository or FTP server as a supplement.

Thanks with regards,

Waley.

**Comment by tmayerguerr** [comment]
Dear Waley

Sebastian, who created the videos, finished his work at the institute last month after successfully completing his dissertation. Unfortunately, he did not keep the scripts. We can unfortunately only provide the videos: [https://ftp.tugraz.at/outgoing/ITSG/groops/videos/groops_tutorial/](https://ftp.tugraz.at/outgoing/ITSG/groops/videos/groops_tutorial/)

regards
Torsten

**Comment by Waley04** [comment]
Hi Professor,

This is noted with thanks.

Regards,

Waley.
